ratelimits/utilities_test.go

```diff
@@ -7,21 +7,21 @@ import (
 )
 
 func TestDomainsForRateLimiting(t *testing.T) {
```

Comment:
It might be clearer to rename this whole test function, too.

Reply:
Good catch.